ykrt/src/mt.rs

```diff
@@ -442,6 +446,18 @@ impl MT {
                     }),
                     Err(e) => todo!("{e:?}"),
                 }
+                #[cfg(tracer_swt)]
+                unsafe {
+                    if !SWT_JUMP_FLAG {
+                        SWT_JUMP_FLAG = true;
+                    } else {
+                        // let func: unsafe fn() = std::mem::transmute(debug_return_into_unopt_cp().as_ptr());
+                        let func: unsafe fn() = std::mem::transmute(RETURN_INTO_UNOPT_CP.as_ptr());
+                        // self.log.log(Verbosity::JITEvent, "returning into unopt cp");
+                        func();
```

Comment:
When I call my control point transition asm code the `$rsp` is affected (I guess by this function execution).
Expected return address at `main+330`: ` 0x7fffffffdcd0`
But $rsp value when `RETURN_INTO_UNOPT_CP` asm is executed is `0x7fffffffd360` which is off by:
`0x7fffffffdcd0 - 0x7fffffffd360  = 2416` 
Therefore I get segfaults downstream...